go/vt/vtgate/engine/mstream.go

```diff
@@ -62,7 +61,7 @@ func (m *MStream) GetTableName() string {
 
 // TryExecute implements the Primitive interface
 func (m *MStream) TryExecute(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable, wantfields bool) (*sqltypes.Result, error) {
-	return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "[BUG] 'Execute' called for Stream")
+	return nil, vterrors.VT13001("TryExecute is not supported for a message stream")
```

Comment:
Make these two errors consistent. Either `a message stream` or `MStream`. I vote for `MStream`.

Reply:
I think let's use `a message stream` as this is easily understandable for someone who is not working on our codebase as @dbussink mentioned in https://github.com/vitessio/vitess/pull/10738#discussion_r1040690729

Reply:
At the same time ... We have been using the engine primitive's name in many other places. Let's use the primitive's name in the error string. These errors are bugs anyway, there is not much the user can do to fix it besides creating an issue.

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)